lib/searchkick/tasks.rb

```diff
@@ -2,7 +2,7 @@
   desc "reindex model"
   task reindex: :environment do
     if ENV["CLASS"]
-      klass = ENV["CLASS"].constantize rescue nil
+      klass = ENV["CLASS"].capitalize.constantize rescue nil
```

Comment:
Perhaps `classify` may be even more useful here than `capitalize`?

Reply:
It depends 🙂I avoided using `classify` so as to support non-Rails apps, but if Searchkick is happily Rails-only then I can change it.

Reply:
Sure, good point! Forgot that it was a Rails only method.